src/nft/components/profile/view/FilterSidebar.tsx

```diff
@@ -181,54 +161,47 @@ const CollectionItem = ({
   }
   return (
     <Row
+      maxWidth="full"
+      overflowX={'hidden'}
+      overflowY={'hidden'}
+      fontWeight="normal"
+      className={`${subheadSmall} ${styles.subRowHover}`}
```

Comment:
non-blocking but you can add all these styles to `subRowHover` to remove a little bit of bloat

Reply:
done ty